models/sap.yml

```diff
@@ -1,11 +1,975 @@
+        description: Rollout in a season
+      - name: profl
+        description: Dangerous goods indicator profile
+      - name: ihivi
+        description: Indicator (highly viscous)
+      - name: iloos
+        description: Indicator (in bulk/liquid)
+      - name: serlv
+        description: Level of explicitness for serial numbers
+      - name: kzgvh
+        description: Packaging material is closed packaging
+      - name: xgchp
+        description: Indicator (Approved batch record required)
+      - name: kzeff
+        description: Assign effectivity parameter values / override change numbers
+      - name: compl
+        description: Material completion level
+      - name: iprkz
+        description: Period indicator for shelf life expiration date
+      - name: rdmhd
+        description: Rounding rule for calculation of SLED
+      - name: przus
+        description: Indicator (Product composition printed on packaging)
+      - name: mtpos_mara
+        description: General item category group
+      - name: bflme
+        description: Generic material with logistical variants
+      - name: nsnid
+        description: NATO item identification number
+
+  - name: sap__0vendor_attr
+    description: Vendor attributs
+    columns:
+      - name: lifnr
+        description: Account number of vendor or creditor 
+      - name: brsch
+        description: Industry key
+      - name: ktokk
+        description: Vendor account group
+      - name: land1
+        description: Country key
+      - name: loevm
+        description: Central deletion flag for master record
+      - name: name1
+        description: Name 1
+      - name: name2
+        description: Name 2
+      - name: name3
+        description: Name 3
+      - name: ort01 
+        description: City
+      - name: ort02 
+        description: District
+      - name: pfach 
+        description: PO Box
+      - name: pstl2 
+        description: P.O. Box postal code
+      - name: pstlz 
+        description: Postal code
+      - name: regio 
+        description: Region (State, province, county)
+      - name: sortl 
+        description: Sort field
+      - name: spras 
+        description: Language key
+      - name: stcd1 
+        description: Tax number 1
+      - name: stcd2 
+        description: Tax number 2
+      - name: stcd3 
+        description: Tax number 3
+      - name: stras 
+        description: Street and house number
+      - name: telf1 
+        description: First telephone number
+      - name: telfx 
+        description: Fax number
+      - name: xcpdk 
+        description: Indicator (is the account a one-time account?)
+      - name: vbund 
+        description: Company id of trading partner
+      - name: kraus 
+        description: Credit information number
+      - name: pfort 
+        description: PO box city
+      - name: werks 
+        description: Plant
+
+  - name: sap__0vendor_text
+    description: Vendor number text
```

Comment:
Request for more details in description of the model.

Reply:
Updated